care/emr/resources/base.py

```
@@ -309,3 +311,44 @@ def delete_model_cache(sender, instance, **kwargs) -> None:
+        all_tag_ids = [tag_id]
+
+    TagConfig.objects.filter(id__in=all_tag_ids).update(cached_parent_json={})
+
+    cache_keys = [
+        model_cache_key(model_string(TagConfig), "TagConfigReadSpec", tid)
+        for tid in all_tag_ids
+    ]
+    cache.delete_many(cache_keys)
+
+
+def invalidate_tag_config_cache(sender, instance, **kwargs):
```

Ruff (0.14.0)

Unused function argument: `sender`

(ARG001)